homeassistant/helpers/config_validation.py

```diff
@@ -884,7 +884,7 @@ def make_entity_service_schema(
                 },
                 extra=extra,
             ),
-            has_at_least_one_key(*ENTITY_SERVICE_FIELDS),
+            has_at_least_one_key(*(str(k) for k in ENTITY_SERVICE_FIELDS)),
```

Comment:
Let's fix this in `has_at_least_one_key` instead.

Reply:
I did consider that, but both `has_at_least_one_key` and `has_at_most_one_key` take `str`s as inputs, and I don't think we ever want to validate non-str keys.

I searched through the code and found one (very similar) usage. (Fixed in a new commit.) All other invocations of `has_at_least_one_key` sends in proper strings.

However, if you (or anyone else) still think these methods should be more flexible, I am happy to do it the other way around.

Reply:
I think the other way around is preferred because it avoids us having this issue. A lot of times voluptuous schemas are passed in, and they have `vol.Optional` and `vol.Required` which behave like strings.

Reply:
OK, behavior is changed!